commit 142c4e0a4e9d85c5d32faf0479b086ee4381502b
Author: q66 <daniel@octaforge.org>
Date:   Fri Sep 16 11:48:52 2022 +0000

    fix build on ppc32
    
    stress-regs.c:120:2: error: void value not ignored as it ought to be

diff --git a/stress-regs.c b/stress-regs.c
index 1ba97a8..63bf3ed 100644
--- a/stress-regs.c
+++ b/stress-regs.c
@@ -117,7 +117,6 @@ static void regs_check128(
 do {								\
 	(void)args;						\
 	(void)reg;						\
-	(void)expected;						\
 	(void)value;						\
 } while (0)
 #endif
